examples/http-server/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+)
+
+func corazaGetRequest(getPath string) (int, string) {
+	if err := setupCoraza(); err != nil {
+		panic(err)
+	}
+	testServer := httptest.NewServer(corazaRequestHandler(http.HandlerFunc(hello)))
+	defer testServer.Close()
+
+	// Performs a get request
+	resp, err := http.Get(testServer.URL + getPath)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	if resp.StatusCode == http.StatusOK {
+		bodyBytes, err := io.ReadAll(resp.Body)
+		if err != nil {
+			log.Fatal(err)
+		}
+		bodyString := string(bodyBytes)
+		return resp.StatusCode, bodyString
+	}
+	return resp.StatusCode, ""
+}
+
+func TestHttpServerTrueNegative(t *testing.T) {
+	expectedStatusCode := 200
+	statusCode, _ := corazaGetRequest("/hello")
+	if statusCode != expectedStatusCode {
+		log.Fatalln(errors.New("Returned status code:" + fmt.Sprint(statusCode) + ". Expected: " + strconv.Itoa(expectedStatusCode)))
```

Comment:
No need to create an error to log a fatal, you can just use the strings. Also, have a look at `log.Fatalf` (see https://pkg.go.dev/log#Fatalf) to format strings: 
```
log.Fatalf("unexpected status code, want: %d, have: %d", statusCode, expectedStatusCode)
```